### PR TITLE
Cep20 mod

### PR DESCRIPTION
**Pull request by FlanFlanagan**
First stab at this. Wasn't sure how to approach a modification of a CEP, but modifying the CEP was what suggested on the last dev call.

### CODE REVIEW

File: source/cep/cep20.rst [hunk 1]
```diff
@@ -178,8 +178,31 @@ with the exception of the core's handling of agent entry/exit
 registration. *Cycamore* modules that deal with agent entry/exit will have to be
 redesigned to incorporate the new execution stack.
 
+Amendment 1. Decision Phase (Post Decommission)
+===================================
```
Comment (scopatz) [inline source/cep/cep20.rst]: FORMATTING: the line underneath needs to be at least as long as the title.

File: source/cep/cep20.rst [hunk 2]
```diff
@@ -178,8 +178,31 @@ with the exception of the core's handling of agent entry/exit
 registration. *Cycamore* modules that deal with agent entry/exit will have to be
 redesigned to incorporate the new execution stack.
 
+Amendment 1. Decision Phase (Post Decommission)
```
Comment (scopatz) [inline source/cep/cep20.rst]: I don't think that this should be its own amendment section, but rather integrated into the text of the document.

**Comment by scopatz** [comment]
Thanks for putting this in @FlanFlanagan! The text looks good to me, but I would like to see it a bit more integrated into the document

**Comment by scopatz** [comment]
Also, I like the name "decision phase"

**Comment by FlanFlanagan** [comment]
Okay! I'll integrate it more. Wasn't sure how to go about approaching it so thanks for the suggestion!

**Comment by FlanFlanagan** [comment]
Modified again. I also moved the Decision phase to directly after the Tock phase as I originally misread the requirements. It's that user phases have to bracket kernal phases, not the other way around. I think this is better anyway as it allows for a decommission condition to occur on the same time step as the decision.

**Comment by FlanFlanagan** [comment]
Ordering of the tock calls is not manageable by the user or agents. Therefore if the institution is agent2 it's decision making will occur before agents spawned by that institution. So any behavior they make during tock will be unknown. 

I shifted it because, I reread the convention and it's just that kernal phases must be surrounded by agent phases. Not the other way around. This positioning also allows users to plan a decommission on the current timestep, as opposed to leaving an agent up for the next time step before the decommission order goes though.

**Comment by gonuke** [comment]
Perhaps this ordering notion could/should be added to the CEP.  It's subtle but important, and easily forgotten, as evidenced by my question....

Given this issue of ordering in any given phase, there is an obvious question about whether you will always need one more phase to deal with the fact that the actions (or decisions) of the previous phase cannot be ordered.  It may be useful to draw a line in a sand here?

**Comment by FlanFlanagan** [comment]
I can add some information that will explain the decision to put it where it is. 

As to your other point, I think you're right. We can continue to add phases on top of phases, we need to draw the line somewhere. I personally think this is the right place to draw it, but I'm open to other ideas.

**Comment by bam241** [comment]
thx @FlanFlanagan

**Pull request merged**